classes/02class/exercises/c02-network08/drii-cavalcanti/c02-network08.txt

```diff
@@ -0,0 +1,15 @@
+1. What is the difference between inbound x outbound?
+   The difference between inboud(ingress) and outbound(egress) is the direction of the traffic based on the point-of-view of the subnets interface. In other words, inbound is the traffic that an instance receives and outbound is the traffic that an instance sends.
+    
+2. What is going to happen by default? Everything allowed or denied?
+   By default, each custom network ACL denies all inbound and outbound traffic until you add rules.
```

Comment:
Can you adjust your answer considering the [ACL rules of the exercise?](https://github.com/devopsacademyau/academy/tree/master/classes/02class/exercises/c02-network08#look-for-the-network-acl-created-for-your-vpc-and-answer-the-following)

Is it going to deny inbound/outbound traffic at the ACL level?